### PR TITLE
Issue-30: fixed suspend/resume schedule issue

### PR DESCRIPTION
**Pull request by ansarars**

### CODE REVIEW

File: hpe3par_sdk/client.py [hunk 1]
```diff
@@ -3962,3 +3962,23 @@ def remoteCopyGroupStatusStoppedCheck(self, remote_copy_group_name):
         :              state
         """
         return self.client.remoteCopyGroupStatusStoppedCheck(remote_copy_group_name)
+
+    def scheduleStatusActiveCheck(self, schedule_name):
```
Comment (farhan7500) [inline hpe3par_sdk/client.py]: Change name as per client review comment

File: hpe3par_sdk/client.py [hunk 2]
```diff
@@ -3962,3 +3962,23 @@ def remoteCopyGroupStatusStoppedCheck(self, remote_copy_group_name):
...
+        """
+        Checks whether schedule status is active or not.
+        :param schedule_name - Schedule name
+        :type schedule_name: str
+        :return: True: If schedule status is 'active'
+        :        False: If schedule status in not 'active'
+        """
+        return self.client.scheduleStatusActiveCheck(schedule_name)
+
+    def scheduleStatusSuspendedCheck(self, schedule_name):
```
Comment (farhan7500) [inline hpe3par_sdk/client.py]: Change name as per client review comment

**Comment by sneharai4** [comment]
LGTM.

**Comment by sneharai4** [comment]
@ansarars  Please add the checks here to parse the result of the command output for showsche which we get from pythonclient to see whether the status is active or suspended.

File: hpe3par_sdk/client.py [hunk 3]
```diff
@@ -3962,3 +3962,31 @@ def remoteCopyGroupStatusStoppedCheck(self, remote_copy_group_name):
         :              state
         """
         return self.client.remoteCopyGroupStatusStoppedCheck(remote_copy_group_name)
+
+    def isScheduleActive(self, schedule_name):
```
Comment (sneharai4) [inline hpe3par_sdk/client.py]: @ansarars How is this helping in reducing code duplicacy? You have simply copied the client changes here in sdk which is not we want.
Please add a getScheduleStatus method in client which returns either active or suspended status and in sdk we will have only one method isScheduleActive which will return True if client returns active status else False(which means schedule status is suspended)
Also, in getScheduleStatus method is client should raise an SSHException if none of the status active/suspended is found from the result of getSchedule().

In ansible, we will simply call the isScheduleActive() of SDK and check whether schedule is active or not.

**Pull request merged**